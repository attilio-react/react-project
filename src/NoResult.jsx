import React from 'react';

class NoResult extends React.PureComponent {
  render () {
    return <div style={{fontSize: 'large'}}>No films found</div>;
  }
}


export {NoResult}


