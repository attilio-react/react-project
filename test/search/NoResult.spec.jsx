import * as React from 'react';
import {mount} from 'enzyme';
import renderer from 'react-test-renderer';


import {NoResult} from '|search/NoResult.jsx';

describe('NoResult rendering', () => {
  it('Should render', () => {
    const 
        component = renderer.create(<NoResult />),
        json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});


