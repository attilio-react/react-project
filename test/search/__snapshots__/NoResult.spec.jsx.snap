// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResult rendering Should render 1`] = `
<div
  style={
    Object {
      "fontSize": "large",
    }
  }
>
  No films found
</div>
`;
